src/main/java/org/example/Department.java

```diff
@@ -0,0 +1,20 @@
+package org.example;
+
+// 部署クラス（部署コード、部署名）
+public class Department {
```

Comment:
クラスの説明はこのようなコメントの書き方にしておくとよいです。
javadocの書き方のルールに則っています。
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format
```suggestion
/**
 *  部署クラス（部署コード、部署名）
**/
public class Department {
```

Reply:
ご指摘ありがとうございます！
対応しました。[8535f3a](https://github.com/YaraiM/JavaChapter4Assignment/pull/1/commits/8535f3a191e3d5593aae5b079f6b2977bbfa6cef)